instructions-for-exchanges.md

```diff
@@ -0,0 +1,54 @@
+# Instructions for Exchanges
+
+This document contains `xud` setup instructions for exchange admins and technical users utilizing Docker.
+
+We use `docker-compose` to package `xud`, `mysql`, `lnd`, `btcd`, `ltcd`, `raiden` & `geth` together to make deploying these services as easy as typing a few commands. All configuration between `xud`, `mysql` and other containers are handled automatically by their `docker-compose` config file.
+
+The goal of this is `docker-compose` being able to lauch `xud` along with its dependencies.
+
+## 1. Prerequisites
+* docker (e.g. apt install docker-compose)
+* docker-compose (e.g. apt install docker-compose)
+
+To install these on e.g. Ubuntu, run:
+```bash
+apt install docker
+apt install docker-compose
+```
+
+## 2. Install
+```bash
+git clone https://github.com/ExchangeUnion/xud -b 1.0.0-prealpha
```

Comment:
Will we create a branch, tag or both for the prealpha?

I would personally prefer a tag so that we could do the cloning like this:
```bash
git clone https://github.com/ExchangeUnion/xud.git
cd xud
git checkout tags/v1.0.0-prealpha
```

Reply:
I'd prefer a tag as well.

Reply:
Agreed and fixed. Pls check.